test/attributes.cpp

```diff
@@ -1377,3 +1377,25 @@ TEST(attributes, logical_xor_op)
 
     EXPECT_EQ(g_logical_xor->get_autob(), logical_xor->get_autob());
 }
+
+TEST(attributes, extractimagepatches_op)
+{
+    FactoryRegistry<Node>::get().register_factory<opset3::ExtractImagePatches>();
+    auto data = make_shared<op::Parameter>(element::i32, Shape{64, 3, 10, 10});
+
+    auto sizes = Shape{3, 3};
+    auto strides = Strides{5, 5};
+    auto rates = Shape{1, 1};
+    // auto padding = string("valid");
```

Comment:
Please, remove

Reply:
I removed the unnecessary comment portions. 
About finalizing the operations, you can use the tensor flow code I used to check the behavior, I am sharing below:

import numpy as np;                                                                                                                                      
import tensorflow as tf;
import sys;
def tprint(a):
    with tf.Session() as sess:  print(a.eval())
n=8;
images = [[[[x * n + y + 1] for y in range(n)] for x in range(n)]]
output = tf.extract_image_patches(images=images,
                       sizes=[1, 3, 3, 1],
                       strides=[1, 5, 5, 1],
                       rates=[1, 2, 2, 1],
                       padding='VALID')
                       #padding='SAME')

print("printing output: ");
print(output)
print("printing tensor: ");
tprint(output);

Reply:
I think the specs are finalized now. Can you accept this "review request",  I can't merge because of this requested review.